Modules/syslogmodule.c

```diff
@@ -135,6 +147,13 @@ syslog_openlog_impl(PyObject *module, PyObject *ident, long logopt,
                     long facility)
 /*[clinic end generated code: output=5476c12829b6eb75 input=8a987a96a586eee7]*/
 {
+    // Since the sys.openlog changes the process level state of syslog library,
+    // this operation is only allowed for the main interpreter.
+    if (!is_main_interpreter()) {
+        PyErr_SetString(PyExc_RuntimeError, "subinterpreter can not use syslog.openlog");
```

Comment:
```suggestion
        PyErr_SetString(PyExc_RuntimeError, "subinterpreter can't use syslog.openlog()");
```

Reply:
AFAIK it is more preferable to use "can not" instead of "can't" in a new code.

Reply:
I have no preference, but I care about consistency. The PR was using "can not" and "can't".